pkgs/build-support/rust/import-cargo-lock.nix

```diff
@@ -195,7 +195,7 @@ let
         echo Found crate ${pkg.name} at $crateCargoTOML
         tree=$(dirname $crateCargoTOML)
 
-        ${python3Packages.python.interpreter} -c 'import sys, shutil; shutil.copytree(sys.argv[1], sys.argv[2], ignore_dangling_symlinks=True)' "$tree" "$out"
+        cp -prvL "$tree" "$out"
```

Comment:
As per this [comment](https://github.com/NixOS/nixpkgs/issues/376590#issuecomment-2669982763), maybe rather use `cp --preserve=mode,ownership,timestamps --recursive --verbose --derefence`.

Reply:
Doesn't this change back to using `-L` will cause the [problem](https://github.com/NixOS/nixpkgs/issues/376590#issue-2810803528) for lighthouse again?

Reply:
`lighthouse` uses `fetchCargoVendor` now

Reply:
I don't want to do the expansion that much.
If others also think it is better, I'll do it.